src/packages/cli/src/Version.ts

```diff
@@ -96,7 +96,7 @@ function slugify(str: string): string {
   return str.toString().toLowerCase().replace(/\s+/g, '-')
 }
 
-// @prisma/cli          : 2.0.0-alpha.473
+// @prisma/cli          : 2.0.0-dev.0
```

Comment:
https://github.com/prisma/prisma/pull/2691/files#r437258192

Reply:
The value in here doesn't matter, as it will be overridden with `2.1.0-dev.1`